### PR TITLE
Numpy111 fixup

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,3 +54,4 @@ extra:
     - pelson
     - rgommers
     - ocefpaf
+    - mvanniekerk
```
Comment (jakirkham) [inline recipe/meta.yaml]: Umm... 😕

**Comment by jakirkham** [comment]
Thanks for doing these Marius :)

File: recipe/meta.yaml (hunk 1 above)
Comment (mariusvniekerk) [reply recipe/meta.yaml]: ugh whoops

**Pull request merged**